MOBILE_CLIENT/IOS/_MEDIUM/IPA_URL_SCHEME_HIJACKING/meta.json

```diff
@@ -14,6 +14,10 @@
     ],
     "OWASP_MASVS_L3": [
       "MSTG_PLATFORM_3"
+    ],
```

Comment:
I don't see any mentions of article 24 and 44 in the findings. 44 is related to data transfer. 24 is related to data processing.